app/controllers/admin/assets_controller.rb

```diff
@@ -17,7 +17,7 @@ def edit
   # PATCH/PUT /works/1.json
   def update
     @asset = Asset.find_by_friendlier_id!(params[:id])
-
+    return unless representative_check_for_unpublish
```

Comment:
This is a confusing way to write it, it makes it look like you are returning from an action method without rendering any response at all, which would be wrong. 

I see when I look there is a render in the method... but it's confusing. 

Also methods that return a boolean result should end in `?`. 

What if we change this to a two-liner that's more clear, also taking an explicit param:

    if would_remove_representative?(work)
       render_refused_remove_representative && return
    end

Reply:
That's great. Will use that pattern throughout- thanks.